client/app/components/packages/attachments.hbs

```diff
@@ -0,0 +1,120 @@
+<h3>
+  Attached Documents
+  <a class="section-anchor" id="uploads" data-magellan-target="uploads"></a>
+</h3>
+
+<p>
+  Please attach the following required documents. (Additional materials that help to explain the proposed project may also be submitted.)  
+</p>
+
+<div class="site-main grid-x grid-margin-x">
+  <div class="cell small-6">
+    REQUIRED FOR ALL PROJECTS:
+    <ul>
+      <li>Official Zoning Sectional Map</li>
+      <li>Project Area Photographs</li>
+      <li>Tax Map(s)</li>
+    </ul>
+  </div>
+  <div class="cell small-6">
+    FOR PROJECTS IN THE NYC COASTAL ZONE:
+  </div>
+  <div class="cell">
+    <fieldset class="fieldset">
+      <legend>
+        <b>
+          Attachments
+        </b>
+      </legend>
+      <ul class="no-bullet">  
+        {{#each this.fileManager.existingFiles as |file idx|}}
+          <li
+          class="text-justify"
+          >
+          <div class="grid-x">
+            <div class="cell small-8">
+              <a
+                data-test-document-name={{idx}}
+              >
+                {{file.name}}
+              </a>
+            </div>
+            <div class="cell small-3 text-right">
+              <small>
+                {{file.timeCreated}}
+              </small>
+            </div>
+            <div class="cell small-1 text-right">
+              <button
+              >
+                <FaIcon @icon="times" @prefix="fas" class="text-red" />
+              </button>
+            </div>
+          </div>
+          </li>
```

Comment:
These chunks of code might make good components of their own! Unclear now though... good to keep it all in one component and then see where things go :)